express-post-json/server.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+let PORT = 8080;
+let app = express();
+let grades = {};
+let nextId = 1;
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  let gradesArray = [];
+  res.json(grades);
+});
```

Comment:
The `GET /api/grades` endpoint should respond with an array of grade objects, not the `grades` object of objects.